src/styles/blocks/header.scss

```diff
@@ -27,9 +27,16 @@
     list-style: none;
     line-height: 60px;
 
+    transition: color 0.3s, transform 0.3s;
```

Comment:
I think you should create an extra variable for transition time cause u use it several times.

Reply:
Agree, add var for duration